anda/tools/copyparty/copyparty.spec

```
@@ -0,0 +1,44 @@
+%global debug_package %nil #needed to avoid "empty %files file" errors
+
+Name:           copyparty
+Version:        1.18.5
+Release:        1%?dist
+Summary:        Portable, featureful, and fast file server 
+URL:            https://github.com/9001/copyparty
+Source0:        %url/archive/refs/tags/v%version.tar.gz
+License:        MIT
+BuildRequires:  python3-devel python3-pip
+BuildRequires:  python3dist(wheel) python3dist(build) python3dist(jinja2)
+BuildRequires:  python3dist(setuptools) python3dist(installer)
+Requires:       python3
+BuildArch:		noarch
+Packager:       Riley Loo <dev@zackerthescar.com>
+
+%description
+Portable file server with accelerated resumable uploads, dedup, WebDAV, 
+FTP, TFTP, zeroconf, media indexer, thumbnails++ all in one file, no 
+(runtime) deps (other than Python itself)
+ 
+%prep
+%autosetup -n copyparty-%version
+
+%build
+python3 -m build
```

It is not recommended to build Python projects using `setup.py` directly in Fedora and doing so is considered outdated.
```suggestion
%pyproject_wheel
```

---

Thanks for the suggestion, fixed in new commit!